### ISSUE TITLE
MD059 False positive when descriptive title is provided separately to non-descriptive link text

### ISSUE THREAD

**Issue by hwoodiwiss** [DavidAnson/markdownlint#1623]
With the addition of MD059, I'm seeing errors on a link that does use one of the prohibited texts, however, the link also has separately provided alt-text that gets rendered as the Title attribute of the `<a>` tag.

 The example in question is:

```md
[here](https://raw.githubusercontent.com/justeattakeaway/httpclient-interception/main/src/HttpClientInterception/Bundles/http-request-bundle-schema.json "JSON Schema for HTTP request interception bundles for use with JustEat.HttpClientInterception.").
```

Which renders to [here](https://raw.githubusercontent.com/justeattakeaway/httpclient-interception/main/src/HttpClientInterception/Bundles/http-request-bundle-schema.json "JSON Schema for HTTP request interception bundles for use with JustEat.HttpClientInterception.").

I think in this case, MD059 wouldn't apply as I believe screen readers should read the link title.

**Comment by nschonni** [comment]
No, here is a random article that explains the difference https://www.accessibilitychecker.org/guides/alt-text/
